packages/manager/overthebox/app/cypress/integration/example.spec.js

```diff
@@ -0,0 +1,40 @@
+describe('OverTheBox', () => {
+  it('Module is standalone', async () => {
```

Comment:
I expect a test name to be written like `it(<verb> <expected behaviour/test description>)` so imo, it should be something like `it('should be a standalone module)`